src/models/vendor.js

```diff
@@ -29,8 +29,9 @@ module.exports = (sequelize, DataTypes) => {
     }
   );
 
-  Vendor.associate = function() {
-    //
+  Vendor.associate = function(models) {
+    this.hasOne(models.VendorIdentifier, { foreignKey: 'flavorId' });
```

Comment:
A vendor can have multiple identifiers, up to one per data supplier.

Reply:
Fixed